### PR TITLE
Ci/356 self hosted runners

### PR DESCRIPTION
**Pull request by Uruael**
Modino-io/modino#356

### Status
- [x] Ready for review
(If not, write either WIP or blocked)

### Summary
Updated reusable workflows to use self hosted runners

### Motivation - All related issues
- [ ] I have linked all related issues
#### Issues: 

https://github.com/Modino-io/modino/issues/356

### Blocked by PRs (_optional_)

https://github.com/Modino-io/modino-devops/pull/24

### Validation
- [x] I have made sure no other tests have to be added or updated before this pull request is merged / created a new issue aimed to update them (link the issue if needed)

Tested via github actions

### Checklist

- [x] I have self-reviewed my changes
- [x] I have made sure all BREAKING CHANGEs were marked accordingly (see [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and [semantic-release](https://github.com/semantic-release/semantic-release))
- [x] I have made sure documentation is up to date or created a new issue aimed to update it (link the issue if needed)
- [x] I have made sure that my changes do not require (or break) configuration of other services, e.g. Kubernetes/Docker-compose or created a new issue aimed at addressing this task
- [ ] No other pull requests have to be merged / issues have to be solved before this one

### Other information (_optional_)

### CODE REVIEW

File: .github/workflows/container-reusable.yaml [hunk 1]
```diff
@@ -70,13 +73,8 @@ jobs:
...
           username: ${{ secrets.REGISTRY_LOGIN }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: install buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
+          erc: false
```
Comment (KrzeQ) [inline .github/workflows/container-reusable.yaml:76]: Spelling error
```suggestion
          ecr: false
```
https://github.com/docker/login-action#customizing

Also it's not used in second file, what is it's purpose?

File: .github/workflows/container-reusable.yaml [hunk 2]
```diff
@@ -87,19 +85,89 @@ jobs:
...
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          build-args: |
-            GITLAB_PACKAGES_TOKEN=${{ secrets.GITLAB_PACKAGES_TOKEN }}
-            GITHUB_USERNAME=${{ secrets.GIT_USERNAME }}
-            GITHUB_TOKEN=${{ secrets.GIT_TOKEN }}
-            GITHUB_PACKAGES_TOKEN=${{ secrets.GIT_TOKEN }}
+            name: |
+              image-arm64
+              image-amd64
```
Comment (KrzeQ) [inline .github/workflows/container-reusable.yaml:173]: Is this action really needed? Won't the default [actions/delete-artifact](https://github.com/marketplace/actions/delete-artifact) be enough?

**Review by KrzeQ — CHANGES_REQUESTED** [review]
Minor error, two questions. Overall looks good 👍

File: .github/workflows/container-reusable.yaml (hunk 1 above)
Comment (Uruael) [reply .github/workflows/container-reusable.yaml:76]: It specifies whether the given registry is ECR ([Amazon ERC](https://aws.amazon.com/ecr/)) By default it's set to auto (in our case it's false). Looking at this again, I think this line can be deleted .

**Review by KrzeQ — APPROVED** [review]
Good 👍  But let's wait for other reviews too so we can merge at the same time

**Pull request merged**